src/styles/utils/variables.scss

```
@@ -0,0 +1,7 @@
+$primary-text-color: #060B35;
+$secondary-text-color: #616070;
+$main-color: #00ACDC;
+$nav-link-color: #060b35;
```

Why did you create the two variables with the same color?
```suggestion
$primary-text-color: #060B35;
$secondary-text-color: #616070;
$main-color: #00ACDC;
```